rollup.config.js

```diff
@@ -1,76 +1,75 @@
-import svelte from 'rollup-plugin-svelte'
-import resolve from 'rollup-plugin-node-resolve'
-import commonjs from 'rollup-plugin-commonjs'
```

Comment:
Just curious why the import changed.

Reply:
It was moved: https://github.com/rollup/rollup-plugin-commonjs#moved